waku/v2/protocol/waku_store/waku_store_types.nim

```diff
@@ -37,7 +39,7 @@ type
     direction*: PagingDirection
 
   HistoryQuery* = object
-    topics*: seq[ContentTopic]
+    contentFilters*: seq[HistoryContentFilter]
```

Comment:
I understood a history query should also now optionally define a pubsub `topic`, or is that for a next step?

Reply:
That is right, it is going to be added in the next step.